drivers/performance_measurement/app/models/performance_measurement/report.rb

```diff
@@ -957,7 +961,7 @@ def can_see_client_details?(user)
           value_calculation: ->(client, variant_name) {
             return true if PERMANENT_DESTINATIONS.include?(client.send("#{variant_name}_so_destination"))
             return true if PERMANENT_DESTINATIONS.include?(client.send("#{variant_name}_es_sh_th_rrh_destination"))
-            return true if PERMANENT_DESTINATIONS_OR_STAYER.include?(client.send("#{variant_name}_moved_in_destination"))
+            return true if client.send("#{variant_name}_stayer")
```

Comment:
`_stayer` is not the same as someone who was a stayer or exited to a permanent destination.  I'm not sure this change is quite right.

Reply:
yeah, stayer doesn't seem to be populated in the current config, I hijacked it. I'll back that out